protocol/messenger_communities.go

```diff
@@ -208,7 +210,7 @@ func (m *Messenger) JoinedCommunities() ([]*communities.Community, error) {
 }
 
 func (m *Messenger) CuratedCommunities() (*communities.KnownCommunitiesResponse, error) {
-	testNetworksEnabled, err := m.settings.TestNetworksEnabled()
+	/*testNetworksEnabled, err := m.settings.TestNetworksEnabled()
```

Comment:
Shall we remove this then?

Reply:
yeah, will remove it and add a comment containing the commit that contained the code before the change so it's easier to revert without having to browse the commit history